### PR TITLE
Fix nav links color

### PR DESCRIPTION
**Pull request by joaojusto**
Old:
screen shot 2015-08-25 at 09 58 46

new: 
screen shot 2015-08-25 at 09 58 58

### CODE REVIEW

**Comment by joaojusto** [comment]
@zamith since you were the one reporting this bug, do you mind reviewing the fix? :P

File: app/_sass/components/_btn.sass [hunk 1]
```diff
@@ -33,6 +33,7 @@ $btn-ticket-text-color: #9f4c00
   color: $color-white
   background-color: transparent
 .Nav--scrolled,
+.Nav--alwaysOpaque,
```
Comment (zamith) [inline app/_sass/components/_btn.sass:36]: Was this class already being applied to the nav?

Comment (joaojusto) [inline app/_sass/components/_btn.sass:36]: Yes it was

Comment (zamith) [inline app/_sass/components/_btn.sass:36]: Strange that it had the class but it did nothing...

**Comment by zamith** [comment]
:+1:

**Pull request merged**